lib/static/modules/static-image-accepter.ts

```diff
@@ -0,0 +1,134 @@
+import PropTypes from 'prop-types';
+import {get} from 'lodash';
+import type {ReporterConfig} from '../../types';
+import {COMMITED, STAGED} from '../../constants';
+import * as localStorage from './local-storage-wrapper';
+
+let isEnabled: boolean | null = null;
+
+interface AcceptableImage {
+    id: string;
+    parentId: string;
+    stateName: string;
+    stateNameImageId: string;
+    commitStatus: null | typeof STAGED | typeof COMMITED;
+    originalStatus: string;
+}
+
+type ImagesById = Record<string, {
+    id: string;
+    status: string;
+    stateName: string;
+    parentId: string;
+    actualImg: {path: string, size: {width: number, height: number}};
+    refImg: {path: string, relativePath?: string, size: null | {width: number, height: number}}
+}>
+
+interface LocalStorageValue {
+    date: string,
+    pathName: string,
+    commitedImages: Record<string, string>
+    }
+
+type CommitPayload = Array<{
+    id: string;
+    stateNameImageId: string;
+    image: string;
+    path: string;
+}>
+
+const STATIC_ACCEPTER_PERSISTENT_LOCAL_STORAGE_KEY = 'static-image-accepter';
+
+export const checkIsEnabled = (config: ReporterConfig['staticImageAccepter'], isGui: boolean): boolean => {
```

Comment:
returns `false` if `static accepter` is turned off, or if it is `gui`, or configuration is invalid